### PR TITLE
Madras JUG data is added

### PR DESCRIPTION
**Pull request by zarub2k**
Madras JUG information is added as per the email

### CODE REVIEW

**Comment by ivargrimstad** [comment]
@zarub2k There are a couple of things here.

1. You need to add a "Sign-off" footer to your commit. See details here: https://accounts.eclipse.org/legal/eca/validation/28090

2. The spec-adopters.yml contains errors. Should be like this:

```
---
adopters:
  projects:
  - name: Madras JUG
     homepage_url: https://www.meetup.com/MadrasJUG/
     logo: mjug.svg
     logo_white: mjug.svg

```
Start with fixing this, and then we will see...

File: data/spec_adopters.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 adopters:
   projects:
+  - name: Madras JUG
+    homepage_url: https://www.meetup.com/MadrasJUG/
+    logo: mjug.svg
+    logo_white: mjug.svg
   
 # Template: Example for creating new specification entry
 #  - name: Jakarta Bean Validation
```
Comment (ivargrimstad) [inline data/spec_adopters.yml]: Sorry, I missed a part. This is how the file should look like:

```
---
adopters:
  projects:
  - name: Jakarta JSON Processing
    directory: specifications/jsonp/ 
    adopters:
    - name: Madras JUG
      homepage_url: https://www.meetup.com/MadrasJUG/
      logo: mjug.svg
      logo_white: mjug.svg
```
You will also need to sign the ECA with the email you are using for your GitHub account. This should be the same as for the Eclipse Foundation account. Log in to https://accounts.eclipse.org/ and check the status of your ECA in the right hand menu.

**Comment by ivargrimstad** [comment]
I have created a new PR to get the Madras JUG added to JSON-P

https://github.com/jakartaee/jakarta.ee/pull/668

So this PR can be closed.

**Comment by zarub2k** [comment]
Closing this since new PR is created by Ivar.

**Pull request closed without merging**